recipe/meta.yaml

```diff
@@ -47,6 +50,19 @@ requirements:
     - scipy
 
 test:
+  source_files:
+    # Test data
+    - boat.png
+    - lena.png           # [linux]
+    # Test suite
+    - test_decomp.py     # [linux]
+    - test_dictLearn.py
+    - test_linalg.py
+    - test_prox.py       # [linux]
```

Comment:
As OpenBLAS on macOS seems to have some sensitive memory issues (segfaults), we skip some of these tests. This should be resolvable with a rebuild of OpenBLAS, but will require more investigation. Fortunately with the test suite simply excluding the test module or test data is sufficient to skip that port. So we skip copying over any of the failing portions on macOS.